include/Instrument.h

```diff
@@ -133,7 +133,7 @@ class LMMS_EXPORT Instrument : public Plugin
 
 protected:
 	// fade in to prevent clicks
-	void applyFadeIn( sampleFrame * buf, const NotePlayHandle * _n );
+	void applyFadeIn(sampleFrame * buf, const NotePlayHandle * _n);
```

Comment:
You could remove the variable names here entirely, but at least remove the underscore in `_n`.